app/UserSite.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class UserSite extends Model
+{
+    /**
+     * @var array
+     */
+    protected $fillable = [
+        'user_id',
+        'site_id',
+        'editable',
+    ];
+
```

Comment:
Cast the editable variable to boolean so it's easy to use and more accurate in the policies.

```suggestion
    /**
     * @var string[]
     */
    protected $casts = [
        'editable' => 'boolean',
    ];
    
```